source/common/upstream/outlier_detection_impl.h

```diff
@@ -255,8 +344,9 @@ class DetectorImpl : public Detector, public std::enable_shared_from_this<Detect
   std::list<ChangeStateCb> callbacks_;
   std::unordered_map<HostSharedPtr, DetectorHostMonitorImpl*> host_monitors_;
   EventLoggerSharedPtr event_logger_;
-  double success_rate_average_;
-  double success_rate_ejection_threshold_;
+
+  std::map<DetectorHostMonitor::SuccessRateMonitorType, std::tuple<double, double>>
```

Comment:
Maybe `absl::flat_hash_map` here too.

Reply:
Thanks for suggestion. Converted std::map to absl::flat_map_hash.